application/listeners/listeners.go

```diff
@@ -172,3 +185,23 @@ func sendPostCreatedNotifications(e events.Event) {
 
 	sendNewPostNotifications(post, users)
 }
+
+func sendNewUserWelcome(user models.User) error {
+	if user.Email == "" {
+		return errors.New("'To' email address is required")
+	}
+
+	msg := notifications.Message{
+		Template:  domain.MessageTemplateNewUserWelcome,
+		ToName:    user.GetRealName(),
+		ToEmail:   user.Email,
+		FromEmail: domain.Env.EmailFromAddress,
+		Data: map[string]interface{}{
+			"appName":      domain.Env.AppName,
```

Comment:
Looks like your email template is expecting firstName as well.

Reply:
great catch, thanks!